include/pybind11/exec.h

```diff
@@ -3,6 +3,8 @@
 
     Copyright (c) 2016 Klemens D. Morgenstern <klemens.morgenstern@gmx.net>
 
+    This code is based on the boost.python implementation.

```

Comment:
Need to specify full boost license here. I just checked, and the license specifically requires this.